# All of the critic costs for a single planning instance.
Header header
float32 start_v
float32 start_w
float32 selected_v
float32 selected_w
int32 selected_idx
CriticCosts[] costs
int32 num_critics
